src/app/Recordings/ActiveRecordingsTable.tsx

```diff
@@ -187,8 +185,8 @@ export const ActiveRecordingsTable: React.FC<ActiveRecordingsTableProps> = (prop
   );
 
   const handleCreateRecording = React.useCallback(() => {
-    routerHistory.push(`${url}/create`);
-  }, [routerHistory, url]);
+    navigate('/recordings/create');
```

Comment:
I would rather keep something more similar to the previous logic where the navigation to the subview is basically just a relative path resolution, and the component doesn't need to know its own absolute path. This looks promising:

https://stackoverflow.com/questions/72683709/react-router-v6-useroutematch-equivalent

Reply:
Or even just https://reactrouter.com/en/main/hooks/use-navigate#optionsrelative

Reply:
Ahh that slipped my mind again. Updated to use relative path now.